spark/src/test/scala/org/apache/spark/sql/delta/DeltaTableCreationTests.scala

```diff
@@ -1464,7 +1464,7 @@ trait DeltaTableCreationTests
     test(s"data source table:partition column name containing $specialChars") {
       // On Windows, it looks colon in the file name is illegal by default. See
       // https://support.microsoft.com/en-us/help/289627
```

Comment:
Could you update, or remove the link. It is not valid anymore.

Reply:
I will probably sweep them all in one batch separately. I believe there are more instances of such links.